workorder/customizable_workorder_de.tpl

```diff
@@ -0,0 +1,550 @@
+{#
+                            ***Begin Custom Options***
+Set any of the options in this section from 'false' to 'true' in order to enable them in the template
+#}
+
+{% set chrome_right_margin_fix = false %}			{# Fixes a potential issue where the right side of receipts are cut off in Chrome #}
+{% set firefox_margin_fix = false %}				{# Fixes issue with margins cutting off when printing on a letter printer on a Mac #}
+
+{% set itemized_hours_labor = false %}				{# Shows time spent on Labor item if time spent is present in the charge #}
+{% set per_line_subtotal = false %}					{# Displays Subtotals for each Sale Line (ex. 1 x $5.00) #}
+{% set employee_name_on_labor_charges = false %}	{# Display Employee name on Labor Charges #}
+{% set make_work_order_number_small = false %}		{# Makes work order number inline with the header #}

```

Comment:
Nit: Could you align these attribute descriptions?

Reply:
Sure thing!